rules/duo_rules/duo_user_anomalous_push.yml

```diff
@@ -0,0 +1,40 @@
+AnalysisType: rule
+Filename: duo_user_anomalous_push.py
+RuleID: DUO.User.Denied.AnomalousPush
+DisplayName: DUO User Auth Denied For Anomalous Push
+Enabled: true
+DedupPeriodMinutes: 15
```

Comment:
@edyesed @calkim-panther @andrea-youwakim - whats our guidance on when dedupe period are used, and not used?

Reply:
We do not have formal guidance. From my perspective, it's contextual on the type of event, likely frequency/burst behavior of the log,  and duration of the threat represented by the alert.